src/CubedSpheres/CubedSpheres.jl

```diff
@@ -109,10 +109,13 @@ end
 ##### Output writing for cubed sphere fields
 #####
 
+using Oceananigans.Fields: compute!
 import Oceananigans.OutputWriters: fetch_output
 
-fetch_output(field::AbstractCubedSphereField, model, field_slicer) =
-    Tuple(fetch_output(face_field, model, field_slicer) for face_field in faces(field))
+function fetch_output(field::AbstractCubedSphereField, model, field_slicer)
+    compute!(field)
```

Comment:
👁️

Reply:
Do we want to call `fetch_output` on `face_field`? This might work:


```julia
function fetch_output(field::AbstractCubedSphereField, model, field_slicer)
    compute_at!(field, time(model))
    return Tuple(slice_parent(field_slicer, face_field) for face_field in faces(field))
end
```

Reply:
Or actually a more robust design is

```julia
function fetch_output(field::AbstractCubedSphereField, model, field_slicer)
    compute_at!(field, time(model))
    return Tuple(slice_parent(get_face(field_slicer, i), face_field) for (i, face_field) in enumerate(faces(field)))
end
```

then we can generalize `FieldSlicer` and add an appropriate `get_face` method for it?